packages/venia-ui/lib/components/Header/cartTrigger.css

```diff
@@ -1,9 +1,29 @@
-.root {
+.root_desktop {
     composes: root from '../clickable.css';
     height: 3rem;
     min-width: 3rem;
 }
 
+.root_mobile {
+    composes: root_desktop;
+    display: none;
+}
+
+.background {
```

Comment:
We've had a few conversations about this in Slack, but I'd like to avoid creating (and naming) elements just for presentational quirks. In this case, since I anticipated this situation, I've been pushing back against the design calling for a background on the trigger. Can we find a different way to highlight the trigger when the dropdown is active?

Reply:
This is still outstanding pending an alternate approach / design.

Reply:
Resolved with a new design in https://github.com/magento/pwa-studio/pull/2494/commits/f13d4fee0341526df2c7333485e8033f65609e44.